omnibus/config/projects/agent.rb

```diff
@@ -98,6 +99,8 @@
     'InstallFiles' => "#{Omnibus::Config.source_dir()}/datadog-agent/dd-agent/packaging/datadog-agent/win32/install_files",
     'BinFiles' => "#{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/DataDog/datadog-agent/bin/agent",
     'EtcFiles' => "#{Omnibus::Config.source_dir()}\\etc\\datadog-agent",
+    'IncludePython2' => "#{with_python_runtime? '2'}",
+    'IncludePython3' => "#{with_python_runtime? '3'}",
```

Comment:
@derekwbrown this seems to rely on the fact that the ruby string interpolation of booleans will match what's done in the `if` test of `source.wxs.erb`, could we simply pass a boolean value and test that boolean instead?